packages/ia-components/sandbox/multimode-image-viewer/multimode-image-viewer.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+
+class MultimodeImageViewer extends Component {
+  
+  render() {
+    return (
+      <div style={{
+              backgroundColor:'black',
+              height:'300px',width:"100%", 
```

Comment:
Also, we try not to do inline styling unless it can not be helped. abstract to a `.less` file

Reply:
sure